lib/admin_credentials.rb

```diff
@@ -68,12 +68,12 @@ def get
       end
     end
   end
- 
+
   class URLKubeconfigCredentials < AdminCredentials
     def get
       # export OPENSHIFT_ENV_OCP4_ADMIN_CREDS_SPEC=file:///~/.kube/config
       if opts[:spec].start_with? 'file:///'
-        path = File.expand_path(opts[:spec].split("file:///")[1])
+        path = File.expand_path(opts[:spec].split("file://")[1])
```

Comment:
@akostadinov any reason to change the local `file:`` directive?

Reply:
https://en.wikipedia.org/wiki/File_URI_scheme

Reply:
Yes. According to that page, if you put a URL `file:///tmp/test.kubeconfig` then you expect an absolute PATH `/tmp/test.kubeconfig`. Without the change it is treated as the relative `tmp/test.kubeconfig` PATH.

If we want to allow relative paths, we need to also update line 75 to make it also allow `file://` (only to slashes) although this is not allowed by specification. WDYT?

Reply:
@akostadinov I think we should stick with a scheme that's valid specification. I don't think relative paths are that big of a deal.

Reply:
I thought you needed them, that's why I suggested we can support such. In any case my change I believe is according to spec.

Reply:
@akostadinov ok, I have no objections then.

Reply:
LGTM, once @xiuwang has verified this PR works, then we can merge it.

Reply:
@akostadinov @pruan-rht Sorry for laying down the steps, the PR works, you can merge it now
Thanks～